xbmc/addons/AddonDatabase.cpp

```diff
@@ -76,6 +76,9 @@ void CAddonDatabase::CreateTables()
 
   CLog::Log(LOGINFO, "create package table");
   m_pDS->exec("CREATE TABLE package (id integer primary key, addonID text, filename text, hash text)\n");
+
+  CLog::Log(LOGINFO, "create system table");
+  m_pDS->exec("CREATE TABLE system (id integer primary key, addonID text)\n");
```

Comment:
our sql abstraction doesn't need `\n` or `;` and even choked on it when using mysql at some point.
I do recommend to put all sql in capitals, not just `CREATE TABLE`: `CREATE TABLE system (id INTEGER PRIMARY KEY, addonID TEXT)`